src/routes/PaymentNotices.tsx

```diff
@@ -3,116 +3,90 @@ import Stack from '@mui/material/Stack';
 import Typography from '@mui/material/Typography';
 import { ButtonNaked } from '@pagopa/mui-italia';
 import { PaymentNotice } from 'components/PaymentNotice';
-import { CardProps } from 'components/PaymentNotice/Card';
+import QueryLoader from 'components/QueryLoader';
 import React from 'react';
 import { useTranslation } from 'react-i18next';
-
-const paymentNoticesMock: Array<CardProps & { id: number }> = [
-  {
-    payee: {
-      name: 'Politecnico di Milano'
-    },
-    id: 1,
-    paymentInfo: 'RATA 1 - Anno Accademico 2023/2024',
-    amount: '171,00 €',
-    expiringDate: '31/01/2099'
-  },
-  {
-    payee: {
-      name: 'Comune di Milano'
-    },
-    id: 2,
-    paymentInfo: 'TARI 2024',
-    amount: '171,00 €',
-    multiPayment: true
-  },
-  {
-    payee: {
-      name: 'Comune di Milano'
-    },
-    id: 3,
-    paymentInfo: 'Violazione CDS Verbale 0123456',
-    amount: '28,70 €',
-    multiPayment: true
-  },
-  {
-    payee: {
-      name: 'Istituto d’Istruzione Superiore con un nome Molto Lungo che può andare su più righe'
-    },
-    id: 4,
-    paymentInfo: 'Iscrizione Anno Accademico 2023/2024',
-    amount: '171,00 €',
-    expiringDate: '31/01/2099'
-  }
-];
+import utils from 'utils';
 
 const updatedDate = new Date();
 
 export const PaymentNotices = () => {
   const { t } = useTranslation();
+  const { data, isError } = utils.loaders.getPaymentNotices();
 
   return (
     <Stack height="100%" justifyContent={{ lg: 'space-between' }} component="main">
-      <Stack gap={5} component="section">
-        <Typography paddingTop={3} variant="h4" component="h1">
-          {t('app.paymentNotices.title')}
-        </Typography>
-        <Stack gap={3}>
+      <QueryLoader
+        // TODO: fallback component of behavior be defined
+        fallback={isError && <PaymentNotice.Error />}
+        queryKey="paymentNotices">
+        <Stack gap={5} component="section">
+          <Typography paddingTop={3} variant="h4" component="h1">
+            {t('app.paymentNotices.title')}
+          </Typography>
+          {data.length > 0 ? (
+            <Stack gap={3}>
```

Comment:
What about creating a new component using liner from 28 to 62 so we could rewrite this ternary in a more elegant and concise way?

I mean something like that:
`data.length > 0 ? <Payments.ok data={data} /> : <Payments.empty />`